tests/Integration/Momento.Sdk.Tests/SetTest.cs

```diff
@@ -616,12 +616,8 @@ public async Task SetSampleAsync_UsesCachedStringSet_HappyPath()
         Assert.True(allValues.SetEquals(limitGreaterThanSetSizeHitValues), $"Expected sample with with limit greater than set size to return the entire set; expected ({String.Join(", ", allValues)}), got ({String.Join(", ", limitGreaterThanSetSizeHitValues)})");
         
         CacheSetSampleResponse limitZeroResponse = await client.SetSampleAsync(cacheName, setName, 0);
-        // TODO: for now the server is returning a MISS for this. We will are updating that behavior and will need to fix this
-        // test accordingly, but this is an edge case that we don't need to block the SDK release on so we can fix the test
-        // once the server behavior changes.
-        Assert.True(limitZeroResponse is CacheSetSampleResponse.Miss, $"Unexpected response: {limitZeroResponse}");
-        // var limitZeroHitValues = ((CacheSetSampleResponse.Hit)limitZeroResponse).ValueSetString;
-        // Assert.True(allValues.SetEquals(limitZeroHitValues), $"Expected sample with with limit zero to return the entire set; expected ({allValues}), got ({limitZeroHitValues})");
+        var limitZeroHitValues = ((CacheSetSampleResponse.Hit)limitZeroResponse).ValueSetString;
+        Assert.True(allValues.SetEquals(limitZeroHitValues), $"Expected sample with with limit zero to return the entire set; expected ({allValues}), got ({limitZeroHitValues})");
```

Comment:
Last I recall, limit=0 was supposed to return found with an empty list of results if the key is in the cache else miss. Did that change?

Reply:
fixed in https://github.com/momentohq/client-sdk-dotnet/pull/548/commits/c20ba25f13278cf2e193d2cb91f280764f916962